src/collective/solr/TODO.rst

```diff
@@ -0,0 +1,3 @@
+Create robot test for
+
+ - wildcard search
```

Comment:
It is. I agreed with @tisto to keep a list of doctest which are not ported to robot tests and this is the only one. I move it over to the robot files itself. It is more visible there I guess.